### ISSUE TITLE
Missing dependency - plone.app.contenttypes 

### ISSUE THREAD

**Issue by djowett** [bluedynamics/bda.plone.productshop#10]
Shouldn't this be a dependency in setup.py?  Without it I get an error on vanilla Plone 4.3.11

     ImportError: No module named contenttypes.behaviors.leadimage

**Comment by djowett** [comment]
Or (perhaps more likely) this product should cope with leadimage behavior being absent

**Comment by rnixx** [comment]
Yes leadimage should be optional. A view uses it if available, thus this can be solved like so:

    from zope.interface import Interface
    try:
        from plone.app.contenttypes.behaviors.leadimage import ILeadImage
    except ImportError:
        class ILeadImage(Interface):
            """Dummy lead image interface if plone.app.contenttypes missing"""

feel free to open a PR :)
